### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by EyasuTeshome**
change the mobile version to the desktop version by using media queries the changes are made on index.html and style.css and included 1 new image for the background

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -482,3 +511,118 @@ form {
...
+  #home {
+    padding: 160px 260px 90px 130px;
+    height: 70vh;
+    text-align: left;
+    border-bottom-left-radius: 20%;
+    background-color: #fff;
+  }
+}
+
+@media screen and (min-width: 567px) {
```
Comment (zieeco) [inline css/style.css]: **[REQUIRED CHANGES]**

- Please, kindly ensure your break-point is not lower than 768px to meet the project requirement.

File: css/style.css [hunk 2]
```diff
@@ -482,3 +511,118 @@ form {
 hr {
   flex: 0 0 100%;
 }
+
+@media screen and (max-width: 566px) {
```
Comment (zieeco) [inline css/style.css]: **[REQUIRED CHANGES]**

- Please, kindly ensure your break-point is not lower than 768px to meet the project requirement.

Comment (zieeco) [reply css/style.css]: **[REQUIRED CHANGES]**

- Please, kindly ensure your hamburger icon shows on the mobile version and the `portfolio, about and contact` doesn't shows on the mobile version

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @EyasuTeshome 


Great work with the desktop version 👏 
You've done well implementing the design, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
*Please, try to ensure your design is not having a horizontal scroll behavior to improve the user experience*

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by omar-labana** [comment]
# **Approved ✔️**

Hi,

Congratulations on completing this project! 👏🏼 🎉

You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**